layouts/community/ergodox/drashna/keymap.c

```diff
@@ -464,3 +465,13 @@ void rgb_matrix_indicators_user(void) {
 }
 
 #endif  // RGB_MATRIX_INIT
+
+
+uint16_t get_tapping_term(uint16_t keycode) {
+    if (keycode >= QK_MOD_TAP && keycode <= QK_MOD_TAP_MAX) {
+        if (mod_config(keycode & 0xf) & MOD_MASK_ALT) {
```

Comment:
@drashna 
Is this meant to match mod-taps for (effective) alt (only)?
I've only tried it quickly, but it appears to not behave like that for me at least.
I think it deserves a comment in the code.. :)
(current code (for moonlander): https://github.com/qmk/qmk_firmware/blob/c48e9f9e77a8f8c722957ef306280014b098758d/keyboards/moonlander/keymaps/drashna/keymap.c#L213-L222)